src/ds/d500/d500-mapping.cpp

```diff
@@ -0,0 +1,252 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2023 Intel Corporation. All Rights Reserved.
+
+#include "d500-mapping.h"
+
+#include <mutex>
+#include <chrono>
```

Comment:
Not sure all of this includes are used,
mutex, chrono, maybe more..

Reply:
Great catch - removed here and in other headers